validator-tests/must-have-owned-elements.html

```diff
@@ -0,0 +1,21 @@
+<html lang="en-US">
+<head><title>Must Have Required Owned Elements</title></head>
+  <body>
+    <!--
+    URL: https://www.w3.org/TR/wai-aria-1.2/#mustContain
+    RULE: "There may be times that required owned elements are missing,
+    * for example, while editing or while loading a data set. When a widget is
+    * missing required owned elements due to script execution or loading,
+    * authors MUST mark a containing element with aria-busy equal to true. For
+    * example, until a page is fully initialized and complete, an author could
+    * mark the document element as busy."
+    -->
+    <div id="while-loading" aria-label="while-loading" role="list">
```

Comment:
aria-busy should be on the parent (while-loading) in this case.

Reply:
Oh interesting. I assumed that the busy should be on the required child element since it's in the required child owned elements. Does that mean in the second case it should have no child elements?

Reply:
Updated to add aria-busy to the parent element